packages/app-store/alby/components/EventTypeAppCardInterface.tsx

```diff
@@ -33,6 +33,15 @@ const EventTypeAppCard: EventTypeAppCardComponent = function EventTypeAppCard({
   const { t } = useLocale();
   const recurringEventDefined = eventType.recurringEvent?.count !== undefined;
 
+  let credentialId;
+
+  if (eventType.teamId) {
+    const teamCredential = app.teams.find((team) => team.teamId === eventType.teamId);
+    credentialId = teamCredential?.credentialId ?? app.userCredentialIds[0];
```

Comment:
For event type apps, I found that `app.userCredentialIds` always has one entry.

Reply:
Should this data come from the server directly?